packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -66,7 +70,7 @@ const nodesCountDeep = [
 // TODO: ADO#7111 Schema should be fixed to enable schema based encoding.
 const factory = new SharedTreeFactory({
 	jsonValidator: typeboxValidator,
-	treeEncodeType: TreeCompressionStrategy.Uncompressed,
+	treeEncodeType: TreeCompressionStrategy.Compressed,
```

Comment:
Is there a need to keep benchmarks for a tree without compressed encoding? Wondering if we should maintain tests for both.